src/main/java/emu/grasscutter/game/world/SceneGroupInstance.java

```diff
@@ -48,6 +48,9 @@ public SceneGroupInstance(SceneGroup group, Player owner) {
 
         this.isCached = false; //This is true when the group is not loaded on scene but caches suite data
     }
+    SceneGroupInstance(){
```

Comment:
Not sure why you add this new constructor that's not used in any part of your changes, also, this shouldn't be a valid constructor, all instances should have an owner and an id to be valid instances, this is saved to DB, remember

Reply:
I had cases with null arrays in the db, when filling from the db it seems to use this constructor.